python/main.py

```diff
@@ -56,32 +58,47 @@ class HelloResponse(BaseModel):
     message: str
 
 
+
 @app.get("/", response_model=HelloResponse)
 def hello():
     return HelloResponse(**{"message": "Hello, world!"})
 
 
 class AddItemResponse(BaseModel):
     message: str
-
+@app.get("/items")
+def get_items():
+    try:
+        with open("items.json", "r") as file:
```

Comment:
I think we will need to check the case when `items.json` file does not exist 🤔 

Also, instead of `open("items.json"...`

We should use the `items_file` variable defined above to reference the file